src/app/room-creation/services/random-names-provider.service.ts

```diff
@@ -1,23 +1,23 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { map, Observable, of, tap } from 'rxjs';
+import { AngularFireFunctions } from '@angular/fire/compat/functions';
 
 @Injectable({ providedIn: 'root' })
 export class RandomNamesProvider {
   private names: string[] = [];
-  private readonly url = 'https://proxy.cors.sh/https://names.drycodes.com/40?nameOptions=funnyWords&combine=2&separator=space';
 
-  constructor(private httpClient: HttpClient) {}
+  constructor(private httpClient: HttpClient, private functions: AngularFireFunctions) {}
 
   initialize(): void {
-    this.httpClient.get<string[]>(this.url)
+    this.functions.httpsCallable('getRandomNames')({})
```

Comment:
Let's save as constant somewhere, sab+me for `getAvatar`

Reply:
Good point, sometimes I don't know what's wrong with me